WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift

```diff
@@ -301,48 +301,81 @@ struct ReaderNotificationKeys {
     // MARK: ActionDispatcher Notification helper
 
     class func dispatchToggleSeenMessage(post: ReaderPost, success: Bool) {
-        if success {
-            dispatchNotice(Notice(title: post.isSeen ? NoticeMessages.seenSuccess: NoticeMessages.unseenSuccess))
-        } else {
-            dispatchNotice(Notice(title: post.isSeen ? NoticeMessages.unseenFail : NoticeMessages.seenFail))
+        var notice: Notice {
+            if success {
+                return Notice(title: post.isSeen ? NoticeMessages.seenSuccess: NoticeMessages.unseenSuccess)
```

Comment:
Nitpicky, but could we add a space before the colon for consistency with other ternary operators, and to help visually distinguish it from method argument declarations?

Reply:
Woops. I mucked it up when I changed it. Will be fixed. Thanks!